### ISSUE TITLE
java.lang.NoSuchMethodError : com/google/common/base/Objects$ToStringHelper in Guava:28.1-jre

### ISSUE THREAD

**Issue by seonghoonkang** [Netflix/zuul#643]
Hello. there
I'm starter using zuul 2.1 API Gateway in first time.

But as soon as I started, I failed run zuul-sample for the below reason.

"Caused by: java.lang.NoSuchMethodError: com.google.common.base.Objects.toStringHelper(Ljava/lang/Object;)Lcom/google/common/base/Objects$ToStringHelper;"

Looking for the cause, it was for the following reasons.
in servo-core:0.7.2 'com.netflix.servo.monitor.MonitorConfig.class'  line number 203

```   
   @Override
    public String toString() {
        return Objects.toStringHelper(this)
                .add("name", name)
                .add("tags", tags)
                .add("policy", policy)
                .toString();
    }
```
com.google.common.base.Objects.toStringHelper() has deprecated and removed in guava:28.1-jre

Please. fix it.

Thank you.

**Comment by seonghoonkang** [comment]
I solved the issue. 

1. Delete file "dependencies.lock"
2.  Changed the servo-core version in zuul> zuul-core> build.gradle file from 0.7.2 to 0.12.21.

**Comment by 2775468731** [comment]
> 我解决了这个问题。
> 
> 1. 删除文件“ dependencies.lock”
> 2. 将zuul> zuul-core> build.gradle文件中的伺服核心版本从0.7.2更改为0.12.21。

Thank you very much

**Comment by rasilang** [comment]
Hi.
Is this issue fixed in Zuul's latest version: `2.2.3.RELEASE`?
